src/main/java/com/strandls/utility/service/UtilityService.java

```diff
@@ -77,4 +80,17 @@ public List<String> createTagsMapping(HttpServletRequest request, String objectT
 	public HomePageData reorderHomePageSlider(HttpServletRequest request, List<ReorderHomePage> reorderHomePage);
 
 	public List<Long> getResourceIds(String phrase, String type, String tagRefId);
+
+	public Map<String, Map<Long, List<GalleryConfig>>> createMiniGallery(HttpServletRequest request, Map<Long, List<GalleryConfig>> miniGalleryData);
```

Comment:
instead of having a `Map<String, Map<Long, List<GalleryConfig>>>` , can you refactor it to make the return type as a simple pojo? Its much simpler to debug and maintain